tck-dist/src/main/asciidoc/concurrency-tck-reference-guide.adoc

```diff
@@ -89,7 +92,7 @@ For the current TCK version, this can be done with either Java SE Version 11 or
 
 === Testing Framework
 To better understand how this TCK works, knowing what testing frameworks are being utilized is helpful.
-Knowledge of how these frameworks operate and interact will help during the project setup.
+Knowledge of how these frameworks operate and interact will help during the project set up.
```

Comment:
I think you updated the wrong occurrence of "setup".  The original usage here was correct, where "setup" is a noun.  In the other case, "... framework we use to setup the test will ...", it is used as a verb and so must be switched to, "... framework we use to set up the test will ..."